searx/engines/hex.py

```diff
@@ -21,6 +21,9 @@
 # engine dependent config
 paging = True
 search_url = "https://hex.pm/api/packages/"
+# Valid values: name inserted_at updated_at total_downloads recent_downloads
+sort_criteria = "recent_downloads"
```

Comment:
Please also add that option to `searx/settings.yml`, most instance maintainers probably only use the configuration file and options that are described in it, and don't read the engine implementation code.

Reply:
I moved the variables to settings.
No point of having it duplicated, I also can't see it in other engines - setting are either en engine file or in settings.